phenol-io/src/main/java/org/monarchinitiative/phenol/io/obo/hpo/HpoDiseaseAnnotationParser.java

```diff
@@ -97,8 +97,7 @@ public HpoDiseaseAnnotationParser(String annotationFile, HpoOntology ontlgy) {
             double frequency = getFrequency(line.getFrequency());
             List<TermId> modifiers = getModifiers(line.getModifierList());
             HpoOnset onset = getOnset(line.getOnsetId());
-            HpoAnnotation tidm =
-              new HpoAnnotation(line.getPhenotypeId(), frequency, onset, modifiers);
+            HpoAnnotation tidm = HpoAnnotation.builder(line.getPhenotypeId()).frequency(frequency).onset(onset).modifiers(modifiers).build();
```

Comment:
I'd split this onto separate lines per method call as then if there is an error the stacktrace will point to the exact method:

```
HpoAnnotation tidm = HpoAnnotation.builder(line.getPhenotypeId())
        .frequency(frequency)
        .onset(onset)
        .modifiers(modifiers)
        .build();
```